planning/plan_execution/src/plan_execution.cpp

```diff
@@ -457,6 +457,8 @@ void plan_execution::PlanExecution::planningSceneUpdatedCallback(const planning_
 
 void plan_execution::PlanExecution::doneWithTrajectoryExecution(const moveit_controller_manager::ExecutionStatus &status)
 {
+  // sync all planning scene updates before continuing
+  planning_scene_monitor_->syncSceneUpdates();
```

Comment:
Same here, I do not see a reason to wait for scene updates after we executed a trajectory.